src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/service/ImportService.kt

```diff
@@ -57,19 +61,19 @@ class ImportService(
 
     fun importReports() = importUnlessLocked(reportingImporter)
 
-    fun calculatePrices(): String {
-        return if (importLocations().second == ImportStatus.IN_PROGRESS) ImportStatus.IN_PROGRESS.name
-        else if (importPrices().second == ImportStatus.IN_PROGRESS) ImportStatus.IN_PROGRESS.name
+    fun spreadsheet(supplierName: String): File? {
+        return if (importLocations().second == ImportStatus.IN_PROGRESS) null
+        else if (importPrices().second == ImportStatus.IN_PROGRESS) null
         else {
+            val supplier = Supplier.valueOf(supplierName.toUpperCase())
             val (reports, status) = importReports()
             if (reports != null) {
                 val calculator = calculatorFactory.calculator(reports.toList())
-                val prices = calculator.standardPrices(Supplier.SERCO)
-                prices.toString()
+                val prices = calculator.standardPrices(supplier)
```

Comment:
Perhaps check (if not already) we actually have some prices in the db to report on...meh,